src/fragment-components/a-inline-loading.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {
+	RadioButtonGroup,
+	RadioButton,
+	InlineLoading,
+	TextInput
+} from 'carbon-components-react';
+import { AComponent } from './a-component';
+import { ComponentInfo } from '.';
+import { css } from 'emotion';
+import image from './../assets/component-icons/inline-loading.svg';
+import {
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj,
+	angularClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+
+export const AInlineLoadingSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<legend className="bx--label">Status</legend>
+		<RadioButtonGroup
+			orientation="vertical"
+			name="status-radio-buttons"
+			valueSelected={selectedComponent.status}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				status: event
+			})} >
+			<RadioButton
+			id="inactive"
+			labelText="Inactive"
+			value="inactive"
+			/>
+			<RadioButton
+			id="active"
+			labelText="Active"
+			value="active"
+			/>
+			<RadioButton
+			id="finished"
+			labelText="Finished"
+			value="finished"
+			/>
+			<RadioButton
+			id="error"
+			labelText="Error"
+			value="error"
+			/>
+		</RadioButtonGroup>
```

Comment:
Follow the indentation pattern from the rest of the code base and put the closing `/>` on the same line.